src/webview/memory-webview-view.tsx

```diff
@@ -225,6 +256,17 @@ class App extends React.Component<{}, MemoryAppState> {
         this.setState(prevState => ({ ...prevState, isFrozen: !prevState.isFrozen }));
     }
 
+    protected requestStoreMemory = (): void => {
+        messenger.sendRequest(storeMemoryType, HOST_EXTENSION, {
+            memoryReference: this.state.memoryReference,
+            count: this.state.count,
+            offset: this.state.offset
+        });
+    };
+
+    protected requestApplyMemory = async (): Promise<void> => {
+        await messenger.sendRequest(applyMemoryType, HOST_EXTENSION, undefined);
+    };
```

Comment:
We haven't used `request` in other cases where we make requests; that it's a request to the plugin side is a bit of an implementation detail.

```suggestion
    protected storeMemory = (): void => {
        messenger.sendRequest(storeMemoryType, HOST_EXTENSION, {
            memoryReference: this.state.memoryReference,
            count: this.state.count,
            offset: this.state.offset
        });
    };

    protected applyMemory = async (): Promise<void> => {
        await messenger.sendRequest(applyMemoryType, HOST_EXTENSION, undefined);
    };
```